app/components/agenda/agenda-header/agenda-actions.js

```diff
@@ -57,7 +60,7 @@ export default class AgendaAgendaHeaderAgendaActions extends Component {
   get canPublishDecisions() {
     return (
       this.currentSession.may('manage-decision-publications') &&
-        this.args.meeting.isFinal &&
+        this.isClosed &&
```

Comment:
Similar to what you do in `app/controllers/agenda/agendaitems/agendaitem.js` in this PR, you can define a `get isFinalMeeting() { ... }` and use that instead of introducing the tracked property `isClosed`. 

Advantage of this approach is that the template will automatically be updated if the relation `this.meeting.agenda` is set somewhere in the app, while this will not be the case with the tracked property `isClosed`.